### ISSUE TITLE
Publish the app's .apk from Google Play and F-Droid to GitHub Releases

### ISSUE THREAD

**Issue by ghost** [johannesjo/super-productivity#3596]
## Problem Statement
You should publish the app's Android package (.apk) on GitHub Releases. Getting the app straight from the developer is the fastest way to be up to date because you don't have to go through Google Play's or F-Droid's review process for updates. 

## Possible Solution
Publish the app's Android package (.apk) to GitHub Releases.

—
Similar issue that was closed by the bot for inactivity: https://github.com/johannesjo/super-productivity/issues/2978

**Comment by domportera** [comment]
yes please - im currently stuck not being able to use mobile bc both are behind the latest desktop release which has a different data version /:

**Comment by TypicalUsername-ai** [comment]
> Maybe adding the Obtainium button would be cool: [fork-maintainers/iceraven-browser#849 (comment)](https://github.com/fork-maintainers/iceraven-browser/issues/849#issue-2815164195)

yeah! my thoughts exactly because thats how i get the app (and am frustrated that FDroid lags behind with releases)

**Comment by TypicalUsername-ai** [comment]
I think since the F-Droid process is already set up and there exists a a [workflow](.github/workflows/build-android.yml) by @johannesjo that only needs fixing I'd say the apk release on github is the best starting point
